packages/components/src/Drawer/Drawer.css

```diff
@@ -41,7 +41,7 @@
 
 .header {
   display: flex;
-  padding: var(--drawer--base-padding);
+  padding: var(--space-small) var(--drawer--base-padding);
```

Comment:
Just want to make sure I'm not missing something.  The figma looks like it's `14px` top and bottom?

<img width="209" alt="Screenshot 2023-01-23 at 2 44 07 PM" src="https://user-images.githubusercontent.com/104797704/214135136-57654a16-1544-4437-9863-6b39c80a8b5a.png">

Reply:
That's the visible space around the title, but when looking at the container element you can see how it maps to small (8) here:

![image](https://user-images.githubusercontent.com/39704901/214146442-01de7ad2-0eb4-4b99-9d3d-fbe1afb5a1d7.png)

Reply:
Ooohhhh, got it! Thanks Chris!